test/test_griduniverse.py

```diff
@@ -212,22 +214,3 @@ def test_raises_if_heroku_wont_start(self, debugger):
             Wrapper.return_value = mock_wrapper
             with pytest.raises(OSError):
                 debugger.run()
-
-
-@pytest.mark.usefixtures('env')
-class TestGriduniverse(object):
```

Comment:
Note: I removed this because it appears to be a duplicate of the test at https://github.com/Dallinger/Griduniverse/blob/c45d6309a10ce5e58d2ba28a871251c262af3552/test/test_griduniverse.py#L168